data/remnant/remnant jobs.txt

```diff
@@ -100,11 +100,11 @@ mission "Remnant: Broken Jump Drive job"
 		dialog phrase "broken jump drive on visit"
 	on complete
 		outfit "Jump Drive (Broken)" -1
-		"remnant: broken jump drive count" ++
+		"remnant: broken Jump Drive count" ++
```

Comment:
```suggestion
		"remnant: broken jump drive count" ++
```

This is a condition, not proper text, so it should remain lower case.

Reply:
Oops. I thought I got all those.